Library/Homebrew/cmd/migrate.rb

```diff
@@ -0,0 +1,29 @@
+require "migrator"
+require "formula_renames"
+
+module Homebrew
+  def migrate
+    raise FormulaUnspecifiedError if ARGV.named.empty?
+
+    ARGV.named.each do |name|
```

Comment:
`ARGV#resolved_formulae` resolves formula by current installation first. So it shouldn't matter what order we use to  resolve formula in `Formulary#factory`, right?